internal/server/server.go

```diff
@@ -0,0 +1,26 @@
+package server
+
+import (
+	"go.uber.org/zap"
+	"log"
+	"net/http"
+	"time"
+)
+
+func Server(addr string, hand http.Handler, sugar *zap.SugaredLogger) {
```

Comment:
написал комент где инициализация логера - лучше брать глобальный логер - он нам будет нужен в любой части кода, иначе придется передавать в каждую функцию
```suggestion
func Server(addr string, hand http.Handler) {
```